File: ios/stereoCamera/stereoCamera/Comm/Commands/Disconnect.swift
//
//  Disconnect.swift
//  stereoCamera
//
//  Created by hallmarklabs on 7/9/18.
//  Copyright © 2018 cody. All rights reserved.
//

import Foundation

class Disconnect : Command
{
    override init()
    {
        super.init()
        cmdtype = CommandTypes.DISCONNECT
        expectsResponse = false
    }
}

File: android/app/src/main/java/com/munger/stereocamera/ip/command/commands/Disconnect.java
package com.munger.stereocamera.ip.command.commands;

import com.munger.stereocamera.ip.command.Command;

public class Disconnect extends Command
{
	public Disconnect()
	{
		super();
		cmdtype = Type.DISCONNECT;
		expectsResponse = false;
	}
}
